$OpenBSD: patch-kmymoney_plugins_onlinetasks_national_CMakeLists_txt,v 1.1 2017/08/24 09:31:35 dcoppa Exp $

commit 6d235f2cd835d5d3e00dee43ed29bdf2c5ca80d5
Author: Ralf Habacker <ralf.habacker@freenet.de>
Date:   Tue May 23 19:52:20 2017 +0200

Fix 'duplicated symbol' compile error on Windows.

Because the whole project uses cmake automoc support additional
calls to automoc4 causes 'duplicated symbol' errors on Windows.

* Unbreaks Ninja builds too

Index: kmymoney/plugins/onlinetasks/national/CMakeLists.txt
--- kmymoney/plugins/onlinetasks/national/CMakeLists.txt.orig
+++ kmymoney/plugins/onlinetasks/national/CMakeLists.txt
@@ -20,7 +20,6 @@ kde4_add_ui_files( konlinetasks_national_SCRS
 )
 
 # Make this plugin unit testable
-automoc4(konlinetasks_national_OBJECTS konlinetasks_national_SCRS)
 add_library( konlinetasks_national_OBJECTS OBJECT ${konlinetasks_national_SCRS} )
 
 # Add plugin which is not allowed in the unit test
